src/cards/Cards.stories.tsx

```diff
@@ -46,7 +46,7 @@ storiesOf("Cards", module)
                 <div className="clr-col-lg-4 clr-col-md-8 clr-col-12">
                     <Card onClick={action("onclick - card image top")}>
                         <CardImage>
-                            <img src={require("./placeholder_350x150.png")} />
+                            <img src={require("./placeholder_350x150.png")} alt="" />
```

Comment:
Cant we add a rule to exclude this change.  No need to add ALTs at every image tag.

Reply:
Disabled alt warning rule now